lib/Dialect/FIRRTL/FIRRTLOps.cpp

```diff
@@ -2011,6 +2012,25 @@ FIRRTLType TailPrimOp::inferReturnType(ValueRange operands,
   return IntType::get(input.getContext(), false, width);
 }
 
+//===----------------------------------------------------------------------===//
+// VerbatimExprOp
+//===----------------------------------------------------------------------===//
+
+void VerbatimExprOp::getAsmResultNames(
+    function_ref<void(Value, StringRef)> setNameFn) {
+  // If the string is macro like, then use a pretty name.  We only take the
+  // string up to a weird character (like a paren) and currently ignore
+  // parenthesized expressions.
+  auto isOkCharacter = [](char c) { return llvm::isAlnum(c) || c == '_'; };
+  auto name = string();
```

Comment:
please rename the operand in the definition.  This looks way too close to constructing a string.

Reply:
Yeah good point. Renamed it to `text`.